members/WIL000468.yaml

```diff
@@ -1,33 +0,0 @@
-bioguide: WIL000468
```

Comment:
Just ooc, why are we removing this?

Reply:
It looks like this target got rid of his contact form, see http://legis.wisconsin.gov/assembly/84/kuglitsch/contact-me-link/

We have other contact methods for him like email, so I'm removing his yamls to messages go to this email address instead